assignments/objects.js

```diff
@@ -49,16 +103,47 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  "name": "Susan",
+  "age": 70,
+  "speak": function () {
+    return `My name is ${this.name}`;
+  },
+  "child": {
+    "name": "George",
+    "age": 50,
+    "speak": function () {
+      return `My name is ${this.name}`;
+    },
+    "grandChild": {
+      "name": "Sam",
+      "age": 30,
+      "speak": function () {
+        return `My name is ${this.name}`;
+      }
+    }
+  }
+}
 
 // Log the parent object's name
 
+console.log(parent.name);
+
 // Log the child's age
+console.log(parent.child.name);
 
 // Log the name and age of the grandchild
 
+console.log(parent.child.grandChild);
```

Comment:
for this one, it would be better to log it this way ==> console.log(parent.child.grandchild.name + " " + parent.child.grandchild.age); as written, this return the 'Object' itself. Well done on the whole project, however!
Grade 3!